daemon/mgr/container.go

```diff
@@ -780,7 +781,7 @@ func (mgr *ContainerManager) createContainerdContainer(ctx context.Context, c *C
 	// set container's LogPath
 	mgr.SetContainerLogPath(c)
 
-	runtime, err := mgr.getRuntime(c.HostConfig.Runtime)
+	runtimeOptions, err := mgr.generateRuntimeOptions(c.HostConfig.Runtime)
```

Comment:
This is the real entrance to create a container. Two situation to goto this method, first, normally create a container from user interface, second, PouchContainer reload containers at restart time.

Inconvenient to store the runtimeOptions to the disk, containerd required a typed pointer, but we would get a map if we reload the value from disk(under restart and reload situation). Thus, we generate options every time.